docs/recipes/testing-with-selenium-webdriverjs.md

```diff
@@ -0,0 +1,131 @@
+# Setting up AVA with Selenium WebDriverJS
+
+This recipe shows you how to use the Selenium WebDriverJS (official JavaScript implementation) with AVA to test web apps.
+
+## Setup
+
+This recipe uses the following packages:
+
+1. [selenium-webdriver](https://www.npmjs.com/package/selenium-webdriver)
+2. [chromedriver](https://www.npmjs.com/package/chromedriver)
+
+Install them with:
+
+```console
+$ npm install selenium-webdriver chromedriver
+```
+
+As part of this recipe, we will use Selenium to verify web searches on [Bing](https://www.bing.com) and [Google](https://www.google.com)
+
+## Test files
+
+Create the following files:
+
+- `./test/bingtest.js`
+- `./test/googletest.js`
+
+In both files, let's first include the packages:
+
+```js
+import test from 'ava';
+import {Builder, By, Key, until} from 'selenium-webdriver';
+
+require('chromedriver');
+```
+
+In the `bingtest.js` file, add the following code, which tests whether searching for `webdriver` on Bing, returns results.
+
+```js
+test('Bing Search', async t => {
+	let keyword = "webdriver";
+	let driver = new Builder().forBrowser("chrome").build();
+	await driver.get("http://www.bing.com");
+	await driver.findElement(By.name("q")).sendKeys(keyword + Key.ENTER);
+	await driver.wait(until.titleIs(keyword + " - Bing"));
+	t.true((await driver.findElements(By.css("#b_content #b_results li"))).length > 0);
+	await driver.close();
+}); 
+```
+
+In the `googletest.js` file, instead of a single test, lets add two tests, one each for the terms 'webdriver' and 'avajs'.
+
+Since we would like to initialize the webdriver before each test, we use the [`beforeEach` and `afterEach`](https://github.com/avajs/ava/blob/master/docs/01-writing-tests.md#before--after-hooks) hooks to setup and teardown the driver respectively. Using these hooks, helps reduce the amount of code we would write in each test()
```

Comment:
```suggestion
Since we would like to initialize the webdriver before each test, we use the [`beforeEach` and `afterEach`](https://github.com/avajs/ava/blob/master/docs/01-writing-tests.md#before--after-hooks) hooks to setup and teardown the driver respectively. Using these hooks, helps reduce the amount of code we would write in each `test()`.
```